snf-cyclades-app/synnefo/userdata/models.py

```diff
@@ -57,9 +58,23 @@ class PublicKeyPair(ProfileModel):
     content = models.TextField(validators=[
         MaxLengthValidator(SSH_KEY_MAX_CONTENT_LENGTH)])
     fingerprint = models.CharField(max_length=100, null=False, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True,
```

Comment:
This change was made after @philipgian suggestion that objects that already exist in the database should have a default value.